### ISSUE TITLE
'NoneType' object has no attribute 'event_team'

### ISSUE THREAD

**Issue by rth** [paris-saclay-cds/ramp-board#313]
Since using https://github.com/paris-saclay-cds/ramp-board/pull/301 (and the latest few added commits on master) I'm getting the following exception. It does look unrelated to that PR though and that's why I'm opening this a separate  issue; 
```
[2019-12-15 13:52:20,086] ERROR in app: Exception on /cedb1c067dca511aa457d0b2164e2a93b445fe73/feature_extractor.py [GET]             
Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app                        
    response = self.full_dispatch_request()
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request           
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception           
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise                       
    raise value
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request           
    rv = self.dispatch_request()
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request                
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/miniconda3/envs/testenv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view           
    return func(*args, **kwargs)
  File "/home/ubuntu/src/ramp-board/ramp-frontend/ramp_frontend/views/ramp.py", line 735, in view_model                               
    submission.name)):
  File "/home/ubuntu/src/ramp-board/ramp-database/ramp_database/tools/frontend.py", line 114, in is_accessible_code                   
    if user == submission.event_team.team.admin:
AttributeError: 'NoneType' object has no attribute 'event_team'
```

A check for `user is not None` is missing.

cc @maikia (btw, if you want to receive notifications about issues here without being pinged, you might want to click one the "Watch" button on the top of the repo)..

**Comment by glemaitre** [comment]
@rth how can I trigger this behaviour?

**Comment by glemaitre** [comment]
Uhm actually now that I read again, it seems the same issue than in #350 which I solved in #354
